packages/flutter/lib/src/cupertino/form_section.dart

```diff
@@ -65,8 +54,14 @@ enum _CupertinoFormSectionType { base, insetGrouped }
 /// If null, defaults to [CupertinoColors.systemGroupedBackground].
 ///
 /// {@macro flutter.material.Material.clipBehavior}
+///
+/// See also:
+///
+///  * [CupertinoFormRow], an iOS-style list tile, a typical child of
+///    [CupertinoFormSection].
+///  * [CupertinoListSection], an iOS-style list section.
 class CupertinoFormSection extends StatelessWidget {
-  /// Creates a section that mimics standard iOS forms.
+  /// Creates a section that mimicks standard iOS forms.
```

Comment:
looks like "mimics" is actually the more normal spelling (though mimicks is right too)

Reply:
Ok, I will change back to mimics.